app/javascript/components/key-performance-indicators/component.jsx

```diff
@@ -180,7 +173,7 @@ const KeyPerformanceIndicators = () => {
 
                 <Grid container spacing={2}>
                   <Grid item className={css.grow} xs={12} md={12}>
-                    <SupervisorToCaseworkerRatio dateRanges={allDateRanges} />
+                    <SupervisorToCaseworkerRatio dateRanges={[]} />
```

Comment:
In the SupervisorToCaseworkerRatio component you can default dataRanges  to []. `Component = ({ dataRanges = []}) => {...` That way you won't need to pass a prop here.

Reply:
Good catch. Done.